sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusRequestFailedDetailsParser.cs

```diff
@@ -22,10 +22,10 @@ public override bool TryParse(Response response, out ResponseError? error, out I
                 {
                     response.ContentStream.Position = 0;
                     var errorContentXml = XElement.Load(response.ContentStream);
-                    XElement detail = errorContentXml.Element("Detail");
+                    XElement? detail = errorContentXml.Element("Detail");
 
                     var message = detail?.Value ?? response.Content.ToString();
-                    Match? match = Regex.Match(detail?.Value, "SubCode=(\\d+)\\.");
+                    Match? match = Regex.Match(message, "SubCode=(\\d+)\\.");
```

Comment:
Is this a bug fix/behavioral change?

Reply:
Looks like a bug fix.   On behalf of Service Bus, I'll aprove this one.